OpenStack/RHEL-OSP/7/CMakeLists.txt

```diff
@@ -1 +1,8 @@
 ssg_build_product("rhel-osp7")
+
+if (SSG_CENTOS_DERIVATIVES_ENABLED)
+    ssg_build_derivative_product("rhel-osp7" "centos" "centos-osp7")
+endif()
+if (SSG_SCIENTIFIC_LINUX_DERIVATIVES_ENABLED)
+    ssg_build_derivative_product("rhel-osp7" "sl" "sl-osp7")
+endif()
```

Comment:
@yuumasato 

> We had them in release last release, 0.1.31

Understood; however, the Makfiles were poorly copied and pasted before the CMake migration. If `sl-osp` and `centos-osp` don't exist but `rhel-osp` does, I don't think that we should move them forward because of previous copy/paste issues.